src/Application/Actions/Hooks/StripeCashBalanceUpdate.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Application\Actions\Hooks;
+
+use MatchBot\Application\Actions\ActionPayload;
+use MatchBot\Application\Notifier\StripeChatterInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Notifier\ChatterInterface;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+class StripeCashBalanceUpdate extends Stripe
+{
+    private ChatterInterface $chatter;
+
+    public function __construct(
+        ContainerInterface $container,
+        LoggerInterface $logger,
+    ) {
+        /**
+         * @var ChatterInterface $chatter
+         * Injecting `StripeChatterInterface` directly doesn't work because `Chatter` itself
+         * is final and does not implement our custom interface.
+         */
+        $chatter = $container->get(StripeChatterInterface::class);
+        $this->chatter = $chatter;
+
+        parent::__construct($container, $logger);
+    }
+
+    /**
+     * @return Response
+     */
+    protected function action(Request $request, Response $response, array $args): Response
+    {
+        $validationErrorResponse = $this->prepareEvent(
+            $request,
+            $this->stripeSettings['connectAppWebhookSecret'],
+            true,
+            $response,
+        );
+
+        if ($validationErrorResponse !== null) {
+            return $validationErrorResponse;
+        }
+
+        $event = $this->event ?? throw new \RuntimeException("Stripe event not set");
+
+        $this->logger->info(sprintf(
+            'Received Stripe Connect app event type "%s" on account %s',
+            $event->type,
+            $event->account,
+        ));
+
+        if (getenv('APP_ENV') === 'production') {
+            return $this->respond($response, new ActionPayload(200));
+        } else {
+            // for now, and outside of production, just send the webhook to slack so we can see if we like it.
+            $this->chatter->send(new ChatMessage(
+                "StripeCashBalanceUpdate in development - recieved webhook from stripe: \n\n" .
+                $event->toJSON()
```

Comment:
I'm setting up the webhook in the test stripe account. Searched for stripe balance and selected all four matching events. Stripe then gives us the following suggested PHP code to use:

I was planning on just listening for the `cash_balance.funds_available` but perhaps we might as well listen to all four here even if we do no more than log some of them.

```php
switch ($event->type) {
  case 'cash_balance.funds_available':
    $cashBalance = $event->data->object;
  case 'customer_cash_balance_transaction.created':
    $customerCashBalanceTransaction = $event->data->object;
  case 'refund.created':
    $refund = $event->data->object;
  case 'refund.updated':
    $refund = $event->data->object;
  // ... handle other event types
  default:
    echo 'Received unknown event type ' . $event->type;
}
```